src/scripts/components/time-slider-range/time-slider-range.tsx

```diff
@@ -29,8 +29,8 @@ const TimeSliderRange: FunctionComponent<Props> = ({
 
     return {
       left: `${tickPosition}%`,
-      backgroundColor: isSelected ? 'white' : undefined, // eslint-disable-line no-undefined
-      transform: isSelected ? 'translateX(-50%) scale(2)' : undefined // eslint-disable-line no-undefined
+      backgroundColor: isSelected ? '#00AE9D' : undefined, // eslint-disable-line no-undefined
```

Comment:
Is it possible to use the color variables here as well?

Reply:
unfortunately not, the stylus variables are not accessible from javascript :(